### PR TITLE
Transition to ltm

### PR DESCRIPTION
**Pull request by Lokathor**
Might as well make the intent official at this point.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,6 +4,22 @@
...
 [![gfx-hal:0.1](https://img.shields.io/badge/gfx--hal-0.1-blue.svg)](https://docs.rs/gfx-hal)
 
+# Project Status
+
+We're in long term maintenance mode!
+
+This teaches you how to use `gfx-hal-0.1.0`, but 99% of the world shouldn't be
+using `gfx-hal` directly in the first place. It's absolutely important to know
+what's going on, but then you should move to a higher level API. Which API?
+Well, right now [rendy](https://github.com/omni-viral/rendy) is your best bet.
```
Comment (termhn) [inline README.md]: should probably also mention `wgpu` (https://github.com/gfx-rs/wgpu` here

**Comment by Lokathor** [comment]
uh... what? I can't parse your error bors :(

Let's hope it was a temporary failure?

bors r+

**Pull request merged**